desc/equilibrium/equilibrium.py

```diff
@@ -177,6 +182,9 @@ def __init__(
         L_grid=None,
         M_grid=None,
         N_grid=None,
+        Lz=0,
```

Comment:
what is Lz Mz Nz? if for the omega resolution using something like L_W M_W N_W may be clearer

Reply:
And add any new stuff to the docstring too

Reply:
Using LW, NW, MW for now.

Reply:
The change of variables will be the very last step because I'd have to do this to all the files in examples. tests/inputs/ etc.
I will add the docstrings though